cmd/launcher/control_windows.go

```diff
@@ -1,3 +1,4 @@
+//go:build windows
```

Comment:
asking out of curiosity, are there 2 different build tag formats now? Or is the new one for something specific?

Reply:
To be honest, I'm not sure. I think we're mid change. AFAICT the current recommendation is that all machine parsable stuff start with `//go:`, and I think it might start mattering in 1.17. So a bunch of the linters seem to be complaining about it now. 

I kinda took the easy path here